packages/safe-core-sdk/scripts/generateTypechainFiles.ts

```diff
@@ -14,17 +14,28 @@ const typeChainDirectoryBuildPath = path.join(__dirname, `../${outDirBuild}`)
 
 // Contract list for which the Typechain files will be generated
 // Will be included in dist/ folder
-const safeContractsPath = '../../node_modules/@gnosis.pm/safe-contracts/build/contracts'
-const openZeppelinContractsPath = '../../node_modules/openzeppelin-solidity/build/contracts'
-const safeContracts = [
-  `${safeContractsPath}/GnosisSafe.json`,
-  `${safeContractsPath}/GnosisSafeProxyFactory.json`,
-  `${safeContractsPath}/MultiSend.json`,
+const safeContractsPath = '../../node_modules/@gnosis.pm/safe-deployments/dist/assets'
```

Comment:
Maybe it's better to use `fs` and go from `__basedir` for these?

Reply:
This will be addresed in a different PR together with the import relative to `src/`